build.md

```diff
@@ -57,6 +57,20 @@ Install the build dependencies.
   * Build the binary.
   * Build the docker image with the binary.
 
+* To build multi-arch images, you need to have `buildx` installed. You can follow these [instructions](https://github.com/docker/buildx#installing).
+Supported architectures are: amd64, arm64, armv7, ppc64le. 
+  * NDM multi-arch images are built using QEMU mode(Note: This can take a lot of time since it runs in an emulated environment). If you're on linux, run :
+    * docker run --rm --privileged linuxkit/binfmt:v0.8
+  * Set the value of envirionment variable of `IMAGE_ORG` based on your user name at Dockerhub or any other registry which supports storing multi-arch images( This is required because multi-arch images cannot be stored locally so we push to the registry after building):  
```

Comment:
```suggestion
  * Set the value of environment variable of `IMAGE_ORG` based on your user name at Dockerhub or any other registry which supports storing multi-arch images (This is required because multi-arch images cannot be stored locally, so `buildx` pushes to the registry after building):  
```